Example/Tests/JSONApiTesting.swift

```diff
@@ -76,33 +70,40 @@ class JSONApiTesting: XCTestCase {
-            relationships.append(PSToOne<Author>(property: &self.author, idProperty: &self.authorId, jsonKey: "author"))
+            relationships.append(PSToOne<Author>(property: &self.author, jsonKey: "author"))
         }
 
     }
+    
+    class MultiAuthorPost: Articles {
+        var authors: [Author] = []
+        // swiftlint:disable:next line_length
+        override public func register(attributes: inout [PSJSONAPIProperty],
+                                      andRelationships relationships: inout [PSJSONAPIProperty]) {
+            
+            super.register(attributes: &attributes, andRelationships: &relationships)
+            let authorRelationship = PSToMany<Author>(property: &self.authors,
+                                                      jsonKey: "authors")
+            relationships.append(authorRelationship)
+            
+        }
+        
+    }
 
+    // swiftlint:disable line_length
     class ArticlesTestData: TestData {
         public static var getTestData: Data {
-            return "{\n  \"data\": {\n    \"type\": \"articles\",\n    \"id\": \"1\",\n    \"attributes\": {\n      \"title\": \"JSON API paints my bikeshed!\",\n      \"body\": \"The shortest article. Ever.\",\n      \"created\": \"2015-05-22T14:56:29.000Z\",\n      \"updated\": \"2015-05-22T14:56:28.000Z\"\n    },\n    \"relationships\": {\n      \"author\": {\n        \"data\": {\"id\": \"42\", \"type\": \"people\"}\n      }\n    }\n  },\n  \"included\": [\n    {\n      \"type\": \"people\",\n      \"id\": \"42\",\n      \"attributes\": {\n        \"name\": \"John\",\n        \"age\": 80,\n        \"gender\": \"male\"\n      }\n    }\n  ]\n}".data(using: String.Encoding.utf8)!
+            return "{\n  \"data\": {\n    \"type\": \"articles\",\n    \"id\": \"1\",\n    \"attributes\": {\n      \"title\": \"JSON API paints my bikeshed!\",\n      \"body\": \"The shortest article. Ever.\",\n      \"created\": \"2015-05-22T14:56:29.000Z\",\n      \"updated\": \"2015-05-22T14:56:28.000Z\"\n    },\n    \"relationships\": {\n      \"author\": {\n        \"data\": {\"id\": \"42\", \"type\": \"people\"}\n      },\n \"authors\": {\n        \"data\": [{\"id\": \"42\", \"type\": \"people\"}, {\"id\": \"43\", \"type\": \"people\"}, {\"id\": \"44\", \"type\": \"people\"}]\n      }\n     }\n  },\n  \"included\": [\n    {\n      \"type\": \"authors\",\n      \"id\": \"42\",\n      \"attributes\": {\n        \"name\": \"John\",\n        \"age\": 80,\n        \"gender\": \"male\"\n      }\n    }\n, {\n      \"type\": \"authors\",\n      \"id\": \"43\",\n      \"attributes\": {\n        \"name\": \"Joe\",\n        \"age\": 80,\n        \"gender\": \"male\"\n      }\n    }\n, {\n      \"type\": \"authors\",\n      \"id\": \"44\",\n      \"attributes\": {\n        \"name\": \"Jack\",\n        \"age\": 80,\n        \"gender\": \"male\"\n      }\n    }\n,  ]\n}".data(using: String.Encoding.utf8)!
```

Comment:
I think it makes more sense for the end user of the library to include a TestObjectFactory pattern, but in testing the actual functionality of the library we need to be able to stub the actual JSON. So maybe we should find a way to remove the TestData protocol from the network manager and only expose it for testing in this sample project.